cds_ils/config.py

```diff
@@ -85,6 +86,11 @@ def _parse_env_bool(var_name, default=None):
     return default
 
 
+def query_parser_and(qstr, extra_params):
```

Comment:
May I suggest to, instead of creating an entire query_parser func, simply create/use a hook that allows changing the extra params?

```
def query_params_modifier(query_params):
    query_params["default_operator"] = "AND"
    return query_params
```
basically a hook to allow modifying any query params.

WDYT?